### ISSUE TITLE
Hits in Context Limit in Preferences

### ISSUE THREAD

**Issue by nrshapiro** [Psychoanalytic-Electronic-Publishing/PEP-Web-User-Interface#502]
I went into my preferences in one of my accounts and noticed that the Search - Hits in Context limit was blank.

So I changed it to 3 and saved.

When I went back into Preferences, it showed this:

![image](https://user-images.githubusercontent.com/4837554/118315069-0591dd80-b4c3-11eb-88e1-2d794a0b09be.png)

I changed it to 5, and this time it stuck.

Seems to be ok now, but not sure why it was blank in the first place, and didn't take the first time I changed it.  Something to watch.

Stage. 
Version
Client: 2021.05.13-7731
Server: 8.6.2/2021.0513.1.Beta2
DataSource: OPAS.AWSProd.2021-05-14
Session: 54868fba-3a84-4917-aa1a-dcede4041c1a

**Comment by nrshapiro** [comment]
@bakerac4 @adistasio 

This EXACT scenario just happened to me again on Production after I pushed settings from Stage to Production, and the Staging code itself for the client to production.  Though its worth noting that the user's settings for this are not stored in the API database, they are on PaDS, so not sure why this would happen.

1) I opened up preferences.  Hits in context limit was Blank.  I changed it to 3
2) It said saving at the bottom of the dialog.  I pressed Done.
3) I opened up preferences again.  It said NaN in the hits in context field, rather than the 3 I changed it to.
4) I changed it to 4, and pressed Done
5) I went into preferences again, and it said 4.
